packages/terra-switch/src/terra-dev-site/doc/example/DisabledSwitch.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react';
+import Switch from 'terra-switch';
+
+export default () => {
+  const [isOn] = useState(true);
```

Comment:
We can remove state here as it goes unchanged.

Reply:
removed.